test/extended/testdata/builds/statusfail-oomkilled.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   resources:
     limits:
-      memory: 10Mi
+      memory: 50Mi
```

Comment:
This is enough RAM for the init containers to complete, but not enough for the image pull to complete with `forcePull: true`.